benchmark/performanceTest/main.go

```diff
@@ -38,6 +39,10 @@ func main() {
 		errMsg := fmt.Sprintf("Failed to parse number of test %s with error:%v\n", os.Args[3], err)
 		panic(errMsg)
 	}
+	var parseFileAccessPatterns bool
+	if numberOfTests == 1 {
```

Comment:
This doesn't seem right to me. Can you add a separate parameter, which comes as an input to performanceTest?

Reply:
+1 I suggest further adding a separate parameter ``parseFileAccess`` to ``make benchmarks`` and have it parsed here